INN/umbrella-caribbean
# Fact box styles and style block setting

## Changes

This pull request makes the following changes:

- Creates `caribbean_group_block_fact_box_styling_init` function to include new css files on the frontend and in the editor to show the fact box styles
- Registers new `Fact Box` block style for the `Group` block
- Adds child theme colors to block color palette

Editor:
![Screen Shot 2019-11-25 at 3 40 26 PM](https://user-images.githubusercontent.com/18353636/69576298-fac74e00-0f99-11ea-8940-e659d6c643c5.png)

Desktop:
![Screen Shot 2019-11-25 at 3 40 34 PM](https://user-images.githubusercontent.com/18353636/69576296-fac74e00-0f99-11ea-9822-ac6283e21834.png)

Mobile:
![Screen Shot 2019-11-25 at 3 40 41 PM](https://user-images.githubusercontent.com/18353636/69576295-fa2eb780-0f99-11ea-80bf-2318930dc39f.png)

## Why



For #17

## Testing/Questions

Features that this PR affects:

- Group block fact box


Questions that need to be answered before merging:

- [ ] Is there an actual good way to create a border color picker for the block?

Steps to test this PR:

1. Edit a post and add a group block
2. Add a heading block to the group block
3. Add a list block to the group block
4. Select "Fact Box" under "Styles" for the parent group block
5. Verify the styling looks ok on the editor/desktop/mobile
